aws-lambda/pygeoapi-serverless-config.yml

```diff
@@ -81,8 +81,9 @@ metadata:
         instructions: During hours of service.  Off on weekends.
         role: pointOfContact
 
-datasets:
+resources:
     obs:
+        type: collection
```

Comment:
Isn't the term collection too generic here? Don't we need a term like "collection-of-vector-features", "collection-of-records", "collection-of-coverages", etc? 
Do we actually require a collection to contain similar things, or can a collection contain both vector-features, records and coverages?

Reply:
as discussed in [gitter](https://gitter.im/geopython/pygeoapi?at=5ea34a52e5ed621d4ddb292c) we can delegate to identify the type of assets in the collection to the backend, and if at some point we need to know the type-of-things in the collection upfront, we can add a another parameter for it. @tomkralidis: "we could also think about adding a type key to the provider. Like a Vector tile provider, OAPIF provider, etc."